changelog.d/20240919_155327_chris_json_serializer.rst

```diff
@@ -0,0 +1,4 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- Added a new data serialization strategy, ``JSONData``, which serializes/deserializes in JSON.
```

Comment:
Consider mentioning that this is for _data_, not code.

Reply:
(I recognize it's in the name of the class, but still helpful to call out, I think.)

Reply:
Agreed, updated